prac_07/languages.csv

```diff
@@ -0,0 +1,6 @@
+Language,Typing,Reflection,Year,PointerArithmetic
+Java,Static,Yes,1995,No
+C++,Static,No,1983,Yes
+Python,Dynamic,Yes,1991,No
+Visual Basic,Static,No,1991,No
+Ruby,Dynamic,Yes,1995,No
```

Comment:
maybe I untderstood the exercise wrong put in Walkthrough example, Modification 1. you were asked to add a new language to the csv file. Do you have a new one here?